src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -320,53 +320,42 @@
 
   describe '#user_and_role' do
     let(:user) { create(:user) }
-    let(:nobody) { create(:user_nobody) }
+    let(:logged_in_user) { create(:user) }
+    let(:anonymous_user) { create(:user_nobody) }
```

Comment:
nobody_user is just the factory name. Though internally we have a user with login name '_nobody_', but to me this shouldn't matter for a test.
We have 2 use cases here, one is logged in users and the other one is anonymous or guest users.